employee/src/main/java/ru/fors/employee/data/controller/EmployeeController.kt

```diff
@@ -1,19 +1,28 @@
 package ru.fors.employee.data.controller
 
+import org.springframework.web.bind.annotation.GetMapping
 import org.springframework.web.bind.annotation.PostMapping
 import org.springframework.web.bind.annotation.RequestBody
 import org.springframework.web.bind.annotation.RestController
 import ru.fors.employee.Employee
 import ru.fors.employee.api.domain.dto.EmployeeWithRoleDto
 import ru.fors.employee.api.domain.AddEmployeeUseCase
+import ru.fors.employee.api.domain.GetFullEmployeesInfoUseCase
+import ru.fors.employee.api.domain.dto.FullEmployeeInfoDto
 
 @RestController
```

Comment:
Давай сделаем базовый мапинг на этот контроллер, соответственно методы тож нужно поменять
```suggestion
@RestController
RequestMapping("/employees")
```

Reply:
сделаю в нект пр